.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: bionic
+env:
+  global:
+    - LOCAL_IMAGE="graph-notebook:local"
+    - DEV_IMAGE="purpleteam/graph-notebook-docker-dev:$TRAVIS_COMMIT"
+    - RELEASE_IMAGE="adevinta/graph-notebook-docker"
+services:
+  - docker
+script:
+  - docker build -t $LOCAL_IMAGE .
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: ./script/push_docker.sh $LOCAL_IMAGE $RELEASE_IMAGE
+    on:
+      tags: true
+  - provider: script
+    skip_cleanup: true
+    script: ./script/push_docker.sh $LOCAL_IMAGE $DEV_IMAGE
+    on:
+      all_branches: true
+      tags: false
```

Comment:
Have you tested if the condition `all_branches: true` has effect when `tags: false`?

From the [Travis documentation](https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on):

> tags: false: deployment is triggered if and only if $TRAVIS_TAG is empty. This also causes the branch condition to be ignored.

I'm not sure if this comment only applies to `branch` or also other conditions.

Reply:
I haven't, it's a good question, I will do it and, if that's the case, I will remove the the all_branches condition.

Reply:
Confirmed, "all_branches: true" is not needed, I'm removing it.

Reply:
Okey, finally it looks it is needed: https://app.travis-ci.com/github/adevinta/graph-notebook-docker/builds/234676824, so I added it again.